dll/win32/shell32/CDefView.cpp

```diff
@@ -2106,7 +2106,8 @@ LRESULT CDefView::OnNotify(UINT uMsg, WPARAM wParam, LPARAM lParam, BOOL &bHandl
                 m_sortInfo.bIsAscending = !m_sortInfo.bIsAscending;
             else
                 m_sortInfo.bIsAscending = TRUE;
-            _Sort();
+            if (_DoFolderViewCB(SFVM_COLUMNCLICK, lpnmlv->iSubItem, 0) != S_OK)
```

Comment:
MSDN https://learn.microsoft.com/en-us/windows/win32/shell/sfvm-columnclick says:
> In response to this notification, you should return S_OK to rearrange the list yourself. To have the system folder view object rearrange the list, return S_FALSE.

Therefore I wonder what would happen, if neither `S_OK` nor `S_FALSE` are returned, but some other value :thinking:

Reply:
The callback is going to return E_NOTIMPL for messages it does not handle.